pkg/scaffold/v1/resource/resource_test.go

```diff
@@ -79,24 +79,19 @@ var _ = Describe("Resource", func() {
 			Expect(instance.Validate().Error()).To(ContainSubstring("kind cannot be empty"))
 		})
 
-		It("should fail if the Kind is not camel cased", func() {
+		It("should succeed if the Kind is camel cased", func() {
```

Comment:
maybe include a negative test here too? something that fails dns1035 label validation, just so that we can see that validation error checks are working as expected.

Reply:
[This test](https://github.com/kubernetes-sigs/kubebuilder/pull/1133) on length and [this one](https://github.com/kubernetes-sigs/kubebuilder/blob/cc8d4ffed8f5d02be97c528998e581a7b8776e13/pkg/scaffold/resource/resource_test.go#L91) on allowed characters cover failed validations. Is there something missing? 🤔